woocommerce/woocommerce
# Add CRUD data store delete

### All Submissions:






### Changes proposed in this Pull Request:



Closes #33438 .

### How to test the changes in this Pull Request:

1. Open your console
2. Create a product attribute `wp.data.dispatch('wc/admin/products/attributes').createProductAttribute({ name: 'what is my purpose' } );`
3. Get the product attributes `wp.data.select('wc/admin/products/attributes').getProductAttributes({});`
4. Note the new attribute's ID.
5. Delete the attribute by ID `wp.data.dispatch('wc/admin/products/attributes').deleteProductAttribute( 5 );`
6. Get the attributes again and note that it has been removed and is now `undefined`.  `wp.data.select('wc/admin/products/attributes').getProductAttributes({});`
7. Also note deletion persistence on `wp-admin/edit.php?post_type=product&page=product_attributes`

### Other information:




### FOR PR REVIEWER ONLY:
